src/java/org/apache/cassandra/config/Config.java

```diff
@@ -325,6 +325,9 @@ public MemtableOptions()
     public volatile int concurrent_materialized_view_builders = 1;
     public volatile int reject_repair_compaction_threshold = Integer.MAX_VALUE;
 
+    // The number of executors to use for building secondary indexes
+    public int concurrent_index_builders = 2;
```

Comment:
@josh-mckenzie Should we make a hot-prop out of this?

Reply:
🤔 I think we should have a JMX extension for it and promote it as a first class citizen in the .yaml. If we end up using this same thread pool for index building for SASI it's probably something that's going to see _a  lot_ more usage than it is today.

I originally kept it as the very primitive "let's just break the scheduling coupling" with the suspicion that https://issues.apache.org/jira/browse/CASSANDRA-11218 would make this work here moot, however now that I think on it a bit more there's no predicting when and if that other work will be completed.